functions/imagemagick/index.js

```diff
@@ -71,8 +72,7 @@ const blurImage = async (file, blurredBucketName) => {
 
     console.log(`Downloaded ${file.name} to ${tempLocalPath}.`);
   } catch (err) {
-    console.error('File download failed.', err);
-    return Promise.reject(err);
+    throw new Error(`File download failed: ${err}`);
```

Comment:
Are we sure this will stringify correctly? Do we not mean to provide the `${err.message}`?